example/skel/test.php

```diff
@@ -1,6 +1,18 @@
 <?php
 
+<<<<<<< HEAD
```

Comment:
Forgot to rebase this properly, but I'm going to delete `example/skel` soon as it's just my testbench at this point